galgebra/mv.py

```diff
@@ -1905,94 +1912,123 @@ def correlation(u, v, dec=3):  # Compute the correlation coefficient of vectors
 
 
 def cross(v1, v2):
```

Comment:
Should we also have type annotation for these functions/methods?

Reply:
In a future PR, sure